examples/simple-platform/configure.adoc

```diff
@@ -0,0 +1,251 @@
+export GITHUB_CLIENT_SECRET=""
+# Key pair for the web auth node
+export PUBLIC_KEY=""
+export PRIVATE_KEY=""
+# Peer list override
+export PORT=8888
+export CONNECT_TO_PEERS="localhost:5025"
+----
+
+. Update the configuration in `web-auth-endpoint.sh`:
++
+--
+[loweralpha]
+. Copy the `Client ID` value from the GitHub browser tab and paste it
+  into the `GITHUB_CLIENT_ID` definition in `web-auth-endpoint.sh`.
+
+. Copy the `Client Secret` value from the GitHub browser tab and paste
+  it into the `GITHUB_CLIENT_SECRET` definition in
+  `web-auth-endpoint.sh`.
+
+. Copy the `publicKey` value from `web-auth-endpoint-keys.json` and paste
+  it into the `PUBLIC_KEY` definition in `web-auth-endpoint.sh`.
+
+. Copy the `privateKey` value from `web-auth-endpoint-keys.json` and paste
+  it into the `PRIVATE_KEY` definition in `web-auth-endpoint.sh`.
+--
+
+. Save the updated `web-auth-endpoint.sh` file.
+
+
+== Update the configuration file for the Accounts app
+
+. Navigate to the Accounts app directory:
++
+[source,bash]
+cd ../packages/app-account
+
+. Open the file `interbit.config.js`. The first few lines of this file
+  are:
++
+[source,js]
+----
+const path = require('path')
+const chainAliases = require('./src/constants/chainAliases')
+const { controlActionTypes } = require('./src/constants/actionTypes')
+
+const PUB_KEY = ''
+const WEB_AUTH_PUB_KEY = ''
+
+const config = {
+    ...
+----
+
+. Copy the `publicKey` value from `platform-deploy-keys.json` and paste
+  it into the `PUB_KEY` definition in `interbit.config.js`.
+
+. Copy the `publicKey` value from `web-auth-endpoint-keys.json` and
+  paste it into the `WEB_AUTH_PUB_KEY` definition in
+  `interbit.config.js`.
+
+. Save the updated `interbit.config.js` file.
+
+
+== Update the configuration file for the Template app
+
+. Navigate to the Template app directory:
++
+[source,bash]
+cd ../interbit-template
+
+. Open the file `interbit.config.js`. The first few lines of this file
+  are:
++
+[source,js]
+----
+const path = require('path')
+const chainAliases = require('./src/constants/chainAliases')
+
+const PUBLIC_KEY = ''
+
+const config = {
+    ...
+----
+
+. Copy the `publicKey` value from `platform-deploy-keys.json` and paste
+  it into the `PUBLIC_KEY` definition in `interbit.config.js`.
+
+. Save the updated `interbit.config.js` file.
+
+== Delete the `platform-deploy` manifest file
+
+The manifest file must be deleted when new Interbit key pairs are
+generated, as we did in our setup. If there is a pre-existing manifest,
+the genesis blocks are not overwritten and our new keys do not work.
+Note that when we generate a new manifest, we create new genesis blocks
+which results in new chain IDs (which are hashes of the genesis blocks).
+
+[source,bash]
+----
+cd ../platform-deploy
+rm platform/interbit.manifest.json
+----
+
+
+== Configuration complete
+
+At this point, all of the apps are configured. See the
+link:run.adoc[next section] to see how to start all four applications.
```

Comment:
Can we specifically say Template app, Accounts app, `platform-deploy` node, and `web-auth-endpoint` node since the last two are not apps?